### PR TITLE
fix: fail if ctx show does not have ctx set

### LINKED ISSUE #2117: okteto context show shouldn't prompt for context creation
**Is your feature request related to a problem? Please describe.**
I think that ‘okteto context show’ should fail there’s no context set in the machine, instead of prompting you to create a context. That could be used as a way to check if a context is set on the machine, for scripting purposes.

**Describe the solution you'd like**
I'd like `okteto context show` to fail with an exit code of -1 if there is no okteto context on the machine.

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>

Fixes #2117

## Proposed changes
- I created a new ctxOption which is raiseCtxNotSet. I think its better to have a default value on every command and only use that new option on `context show`
-

### CODE REVIEW

File: pkg/errors/errors.go [hunk 1]
```diff
@@ -48,6 +48,9 @@ var (
 	// ErrNotLogged is raised when we can't get the user token
 	ErrNotLogged = "your token is invalid. Please run 'okteto context use %s' and try again"
 
+	// ErrCtxNotSet is raised when we don't have ctx set
+	ErrCtxNotSet = fmt.Errorf("your context is not set. Please run 'okteto context use' and and select your context")
```
Comment (pchico83) [inline pkg/errors/errors.go:52]:
```suggestion
	ErrCtxNotSet = fmt.Errorf("your context is not set. Please run 'okteto context' and and select your context")
```

**Review by pchico83 — APPROVED** [review]

File: pkg/errors/errors.go (hunk 1 above)
Comment (rberrelleza) [reply pkg/errors/errors.go:52]: There's a double `and and` in the suggestion.

Comment (rberrelleza) [inline pkg/errors/errors.go:52]:
```suggestion
	ErrCtxNotSet = fmt.Errorf("your context is not set. Please run 'okteto context use' to set the default context")
```

**Review by RinkiyaKeDad — APPROVED** [review]

File: cmd/context/cmd_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Okteto Authors
```
[the remark this answers is not in the document]
Comment (pchico83) [reply cmd/context/cmd_test.go:1]: @RinkiyaKeDad I created this issue to track it

**Review by pchico83 — APPROVED** [review]

**Pull request merged**